src/abstractdataframe/selection.jl

```diff
@@ -434,8 +445,17 @@ select(df::DataFrame, c::Union{AbstractVector{<:Integer}, AbstractVector{Symbol}
 select(df::DataFrame, c::ColumnIndex; copycols::Bool=true) =
     select(df, [c], copycols=copycols)
 
-select(df::DataFrame, cs...; copycols::Bool=true) =
-    _select(df, [normalize_selection(index(df), c) for c in cs], copycols)
+function select(df::DataFrame, cs...; copycols::Bool=true)
+    cs_vec = []
+    for v in cs
```

Comment:
Actually I realize that `reduce(vcat, cs)` seems to have the same effect since it doesn't treat pairs as collections. Am I right? Also valid for the `combine` PR.

Reply:
Unfortunately not as we do not want to unwrap vectors other than `AbstractVector{<:Pair}`:
```
julia> vcat([:a, :b], [1,2])
4-element Array{Any,1}:
  :a
  :b
 1
 2
```
and we want:
```
julia> [[:a, :b], [1,2]]
2-element Array{Array{T,1} where T,1}:
 [:a, :b]
 [1, 2]
```
Also `vcat` would lead to strange uncontrolled things like:
```
julia> vcat(:a, [:a=>:b :c=>:d])
2×2 Array{Any,2}:
    :a      :a
 :a=>:b  :c=>:d
```
that would get through and produce strange errors later (we could check for this, but for me the code I proposed was simpler, also I make `cs_vec` of eltype `Any` to save a bit of compilation cost).